source/guides/assets/overview.md

```diff
@@ -0,0 +1,250 @@
+      assets do
+        # digest false, disabled by default
+      end
+    end
+
+    configure :production do
+      digest true
+    end
+  end
+end
+```
+
+If enabled, [assets helpers](/guides/helpers/assets) will generate checksum relative URLs.
+
+```erb
+<%= javascript 'application' %>
+```
+
+```html
+<script src="/assets/application-d1829dc353b734e3adc24855693b70f9.js" type="text/javascript"></script>
+```
+
+## Serve Static Assets
+
+It can dinamically serve them during development.
+It mounts `Lotus::Static` middleware in project Rack stack. This this component is conditionally activated, if the environment variable `SERVE_STATIC_ASSETS` equals to `true`.
+
+By default, new projects are generated with this feature enabled in _development_ and _test_ mode, via their corresponding `.env.*` files.
+
+```
+# .env.development
+# ...
+SERVE_STATIC_ASSETS="true"
+```
+
+Lotus assumes that projects in _production_ mode are deployed using a web server like Nginx that is responsible to serve them without even hitting the Ruby code.
+
+<p class="convention">
+  Static assets serving is enabled by default in <em>development</em> and <em>test</em> environments, but turned off for <em>production</em>.
+</p>
+
+There are cases where this assumption isn't true. For instance, Heroku requires Ruby web apps to serve static assets.
+To enable this feature in production, just make sure that this special environment variable is set to `true` (in `.env` or `.env.production`).
+
+### What Does It Mean To Serve Static Assets With Lotus?
+
+As mentioned above, when this feature is enabled, a special middleware is added in front of the project Rack stack: `Lotus::Static`.
+
+Incoming requests can generate the following use cases
+
+#### Fresh Asset
+
+```
+GET /assets/application.js
+```
+
+It copies the `apps/web/assets/javascripts/application.js` to `public/assets/application.js` and then serves it.
+
+<p class="notice">
+  Assets are copied only if the destination path is NOT existing (eg. <code>public/assets/application.js</code>).
+  If it DOES exist, the asset is only served, without copying it.
+</p>
+
+<p class="warning">
+  When an application has turned OFF assets compilation (Compile mode), Lotus won't copy the file.
+</p>
+
+#### Stale Asset
+
+This could happen in _development_ mode, when we require an asset the first time, it's get copied over `public/`, then we edit it, so the destination file is stale.
+
+```
+GET /assets/application.js
+# edit the original file: apps/web/assets/javascripts/application.js
+# then require it again
+GET /assets/application.js
+```
+
+It copies **again** the source into the destination file (`public/assets/application.js`) and then serves it.
+
+#### Precompiled Asset
+
+Let's say we use Sass to write our stylesheets.
+
+```
+GET /assets/application.css
+```
+
+It preprocess the `apps/web/assets/stylesheet/application.css.sass` to `public/assets/application.css` and then serves it.
+
+#### Dynamic Resource
```

Comment:
i wanted to say if the user needs to know the difference between a request to static file and a resource